src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -190,12 +208,14 @@ public int instanceMain(final String[] argv) {
 
             // Output a one liner about who/where and what software/os we're running on
             try {
-            System.err.println("[" + new Date() + "] Executing as " +
-                                       System.getProperty("user.name") + "@" + InetAddress.getLocalHost().getHostName() +
-                                       " on " + System.getProperty("os.name") + " " + System.getProperty("os.version") +
-                                       " " + System.getProperty("os.arch") + "; " + System.getProperty("java.vm.name") +
-                                       " " + System.getProperty("java.runtime.version") +
-                                       "; Picard version: " + commandLineParser.getVersion());
+                final String msg = String.format(
+                    "[%s] Executing as %s@%s on %s %s %s; %s %s; Deflater: %s; Inflater: %s; Picard version: %s",
+                    new Date(), System.getProperty("user.name"), InetAddress.getLocalHost().getHostName(),
+                    System.getProperty("os.name"), System.getProperty("os.version"), System.getProperty("os.arch"),
+                    System.getProperty("java.vm.name"), System.getProperty("java.runtime.version"),
+                    USE_JDK_DEFLATER ? "Jdk" : "Intel", USE_JDK_INFLATER ? "Jdk" : "Intel",
```

Comment:
You need to test whether the Intel Deflater/Inflater were actually successfully loaded, and use that result to inform the user which deflater/inflater will be used. What you want to do here is:

```
        final boolean usingIntelDeflater = (BlockCompressedOutputStream.getDefaultDeflaterFactory() instanceof IntelDeflaterFactory && ((IntelDeflaterFactory)BlockCompressedOutputStream.getDefaultDeflaterFactory()).usingIntelDeflater());
        final boolean usingIntelInflater = (BlockGunzipper.getDefaultInflaterFactory() instanceof IntelInflaterFactory && ((IntelInflaterFactory)BlockGunzipper.getDefaultInflaterFactory()).usingIntelInflater());
```